examples/acme/manage_acme_docker_environment.sh

```diff
@@ -0,0 +1,101 @@
+#! /usr/bin/env bash
+
+# Initialize variables
+readonly traefik_url="traefik.localhost.com"
+readonly basedir=$(dirname $0)
+readonly doc_file=$basedir"/compose-acme.yml"
+
+# Stop and remove Docker environment
+down_environment() {
+    echo "STOP Docker environment"
+    ! docker-compose -f $doc_file down -v &>/dev/null && \
+        echo "[ERROR] Impossible to stop the Docker environment" && exit 11
+}
+
+# Create dans start (up) environment of subpart of its services
```

Comment:
s/dans/and
`environment of subpart of its services` not sure to understand this part